client/src/components/MultiJurisdictionAudit/AASetup/Review/index.tsx

```diff
@@ -139,6 +139,58 @@ const Review: React.FC<IProps> = ({
     isFileProcessed(j.ballotManifest)
   ).length
 
+  const getBatchTotal = () =>
+    contests
```

Comment:
why do you need to check `contests`? isn't it guaranteed to be defined when this function is called?